certified-connectors/Projectplace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1190 @@
+                }
+              },
+              "required": [
+                "card_id"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/v2/board/{board_id}/cards": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "ok",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "id"
+                  },
+                  "title": {
+                    "type": "string",
+                    "description": "title"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "get_cards",
+        "description": "get_cards",
+        "operationId": "get_cards",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "board_id",
+            "in": "path",
+            "required": true,
+            "type": "integer",
+            "format": "int32"
+          }
+        ]
+      }
+    },
+    "/v1/external_notifications/lists/list_boards": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "ok",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "board": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "board"
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "name"
+                  },
+                  "project_id": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "project_id"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "list_boards",
+        "description": "list_boards",
+        "operationId": "list_boards",
+        "parameters": [],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/v1/external_notifications/{board_id}/list_planlets": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "ok",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "id"
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "name"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "get_planlets",
```

Comment:
@ygavish This operation may not be visible to public, but still the summary and description values should match the standards, please provide a valid sentence.

Refer [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)